test/AutoRest.TestServer.Tests/validation.cs

```diff
@@ -35,5 +35,11 @@ public void ConstructorsCheckRequiredProperties()
             Assert.Throws<ArgumentNullException>(() => new Product(null, new ConstantProduct()));
             Assert.Throws<ArgumentNullException>(() => new Product(new ChildProduct(), null));
         }
+
+        [Test]
+        public void ThrowsIfApiVersionIsNull()
+        {
+            Assert.Throws<ArgumentNullException>(() => new ServiceClient(ClientDiagnostics, null, string.Empty, null, null));
```

Comment:
Which parameter is being checked for null here?

Reply:
Last parameter i.e. `apiVersion`. Replaced other params with random values to make this clear.